webapp/src/components/App/Singlestudy/explore/Modelization/Areas/Hydro/Allocation/AllocationField.tsx

```diff
@@ -0,0 +1,76 @@
+import { Typography, IconButton, Box } from "@mui/material";
+import RemoveCircleOutlineIcon from "@mui/icons-material/RemoveCircleOutline";
+import NumberFE from "../../../../../../../common/fieldEditors/NumberFE";
+import { useFormContextPlus } from "../../../../../../../common/Form";
+import { AllocationFormFields } from "./utils";
+import { getCurrentAreaId } from "../../../../../../../../redux/selectors";
+import useAppSelector from "../../../../../../../../redux/hooks/useAppSelector";
+
+interface Props {
+  field: {
+    id: string;
+    areaId: string;
+    coefficient: number;
+  };
+  index: number;
+  label: string;
+  remove: (index: number) => void;
+}
+
+function AllocationField({ field, index, label, remove }: Props) {
+  const { control } = useFormContextPlus<AllocationFormFields>();
+  const currentAreaId = useAppSelector(getCurrentAreaId);
+
+  ////////////////////////////////////////////////////////////////
+  // JSX
+  ////////////////////////////////////////////////////////////////
+
+  return (
+    <Box
+      key={field.id}
+      sx={{
+        display: "flex",
+        alignContent: "center",
+        alignItems: "center",
+      }}
+    >
+      <Typography
+        sx={{
+          width: "20%",
```

Comment:
Why 20% only?

Reply:
looking at it again I think it's better with a Grid layout, will update this